tests/apps/madeline/test_madeline.py

```diff
@@ -0,0 +1,80 @@
+"""Tests for CGs madeline app"""
+from cg.apps.madeline import make_ped, run
+
+
+def test_run_madeline(madeline_process, madeline_input):
+    # GIVEN a ped stream and a madeline process mock
+    # WHEN running the madeline command
+    outpath = run(madeline_process, madeline_input)
+    # THEN assert a madeline xml file is returned
+    assert outpath.endswith(".xml")
+
+
+def test_generate_madeline_input_no_mother(madeline_columns, proband):
+    # GIVEN a family id and a ind with unknown sex
+    family_id = "test"
+    proband.pop("mother")
+    inds = [proband]
+    # WHEN generating madeline formated lines
+    madeline_lines = make_ped(family_id, inds)
+    i = 0
+    for i, line in enumerate(madeline_lines, 1):
+        if i == 1:
+            continue
+        ind_info = dict(zip(madeline_columns.keys(), line.rstrip().split("\t")))
```

Comment:
Should this be in a method or something, seems like we do this several times in the tests atleast